### PR TITLE
Added NOTNode and implemented getters and setters for nodes and graphs

### PR DESCRIPTION
**Pull request by vismate**

### CODE REVIEW

**Comment by vismate** [comment]
With this merged #22 and #23 can be closed

File: app/src/main/java/ELME/Model/Graph.java [hunk 1]
```diff
@@ -31,4 +31,21 @@ public void evaluateImpl() {
...
 
+    public ArrayList<Node> getNodes() {
+        return nodes;
+    }
+
+    public void setNodes(ArrayList<Node> nodes) {
+        this.nodes = nodes;
+    }
+
+    public boolean isIsCollapsed() {
```
Comment (AMIOL7) [inline app/src/main/java/ELME/Model/Graph.java]: IsCollapsed should be enough for the name of the method

**Review by AMIOL7 — APPROVED** [review]
Just a small comment. Otherwise, you are free to merge

File: app/src/main/java/ELME/Model/Graph.java (hunk 1 above)
Comment (vismate) [reply app/src/main/java/ELME/Model/Graph.java]: Oh, yepp. Will correct that. Pesky auto-generation....

**Comment by vismate** [comment]
Made 2 small fixes, please review and merge!
@pszi

**Review by pszi — APPROVED** [review]
Odd that I didn't see this somehow, but my bad, sorry.

**Pull request merged**